### PR TITLE
Link to the new roadmap at dash.org

### PR DESCRIPTION
**Pull request by nmarley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,3 @@
...
-<br /><a name="f27">27</a>: Using third party API keys, allowing user to setup keys in settings then use from wallet seamlessly without ever leaving Dash.
-<br /><a name="f28">28</a>: By leveraging dapi and third party tools, we can deploy addons and a marketplace, which allows custom third party services to integrate directly in the wallet such as debit/credit cards and fiat bank account support
-<br /><a name="f29">29</a>: High-end quality processor like the Xeon, multi-core. Recommend to at least have 8 cores.
-<br /><a name="f30">30</a>: Sporks, our software based forking mechanism will be controlled by a few members of the core-team. Three of five keys will be used as a threshold for activation of specific core-daemon features. This is to stop accidents happening, damaging the network due to neglect or misunderstanding and outright network abuse.
+The Official Dash Roadmap can be found at https://www.dash.org/roadmap/. Previous versions can be found in the history of this Git repository.
```
Comment (UdjinM6) [inline README.md]: > the history of this Git repository

Let's maybe provide a direct link for non-technical people?
https://github.com/dashpay/dash-roadmap/tree/eb7f6bf4c0700fd2568a22b9a0b97881c55fe4da (the most recent one)

Comment (nmarley) [reply README.md]: Yeah, I agree. Or a link to the commits.

**Comment by nmarley** [comment]
@UdjinM6 How does it look now?

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**